pulpcore/cli/core/repository.py

```diff
@@ -32,3 +47,49 @@ def repository(ctx: click.Context, pulp_ctx: PulpCLIContext) -> None:
 
 repository.add_command(list_command(decorators=filter_options))
 repository.add_command(version_command(decorators=[], list_only=True))
+
+
+@repository.command()
+@pass_repository_context
+@pass_pulp_context
+@repositories_option
+@pulp_option(
+    "--all",
+    "all_repos",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help=_("Select all repositories."),
+)
+@click.option(
+    "--keep-version",
+    "keep_versions",
+    type=tuple([str, int]),
+    help=_("List of (pulp hrefs) repository versions to keep."),
```

Comment:
This is not true.

Reply:
Another leftover from previous state of the PR, thanks for the heads up